dist/css/boosted-grid.css

```diff
@@ -14,224 +14,9 @@
-  --bs-primary-text-rgb: 255, 121, 0;
-  --bs-link-color: #fff;
-  --bs-link-hover-color: #ff7900;
-  --bs-link-color-rgb: 255, 255, 255;
-  --bs-link-hover-color-rgb: 255, 121, 0;
-  --bs-focus-visible-inner-color: #000;
-  --bs-focus-visible-outer-color: #fff;
-  --o-caption-color: #fff;
-  --bs-code-color: #999;
-  --bs-highlight-color: #000;
-  --bs-highlight-bg: #fff;
-  --o-kbd-color: #fff;
-  --o-kbd-bg: #333;
-  --o-pre-color: #eee;
-}
-[class*=bg-]:not([class*=bg-black],
-[class*=-dark]:not(.border-dark):not(.text-dark):not(.btn-dark),
-[class*=bg-secondary]):not(.bg-transparent) {
-  --bs-primary-text-rgb: 241, 110, 0;
-  --bs-link-color: #000;
-  --bs-link-hover-color: #f16e00;
-  --bs-link-color-rgb: 0, 0, 0;
-  --bs-link-hover-color-rgb: 241, 110, 0;
-  --bs-focus-visible-inner-color: #fff;
-  --bs-focus-visible-outer-color: #000;
-  --o-caption-color: var(--o-caption-color, #000);
-  --bs-code-color: #666;
-  --bs-highlight-color: #fff;
-  --bs-highlight-bg: #000;
-  --o-kbd-color: var(--o-kbd-color, #000);
-  --o-kbd-bg: var(--o-kbd-bg, #eee);
-  --o-pre-color: var(--o-pre-color, #333);
-}
-
-[data-bs-theme=dark] {
-  --bs-body-color: #ccc;
-  --bs-body-color-rgb: 204, 204, 204;
-  --bs-body-bg: #333;
-  --bs-body-bg-rgb: 51, 51, 51;
-  --bs-emphasis-color: #fafafa;
-  --bs-emphasis-color-rgb: 250, 250, 250;
-  --bs-secondary-color: rgba(204, 204, 204, 0.75);
-  --bs-secondary-color-rgb: 204, 204, 204;
-  --bs-secondary-bg: #595959;
-  --bs-secondary-bg-rgb: 89, 89, 89;
-  --bs-tertiary-color: rgba(204, 204, 204, 0.5);
-  --bs-tertiary-color-rgb: 204, 204, 204;
-  --bs-tertiary-bg: #464646;
-  --bs-tertiary-bg-rgb: 70, 70, 70;
-  --bs-emphasis-color: #fff;
-  --bs-primary-text: #97b2e9;
-  --bs-secondary-text: #eee;
-  --bs-success-text: #84de84;
-  --bs-info-text: #93d2f0;
-  --bs-warning-text: #ffe066;
-  --bs-danger-text: #e18a72;
-  --bs-light-text: #fafafa;
-  --bs-dark-text: #eee;
-  --bs-primary-bg-subtle: #10192c;
-  --bs-secondary-bg-subtle: #333;
-  --bs-success-bg-subtle: #0a280a;
-  --bs-info-bg-subtle: #0f242e;
-  --bs-warning-bg-subtle: #332900;
-  --bs-danger-bg-subtle: #290c04;
-  --bs-light-bg-subtle: #595959;
-  --bs-dark-bg-subtle: #2d2d2d;
-  --bs-primary-border-subtle: #314c83;
-  --bs-secondary-border-subtle: #666;
-  --bs-success-border-subtle: #1e781e;
-  --bs-info-border-subtle: #1e485c;
-  --bs-warning-border-subtle: #665200;
-  --bs-danger-border-subtle: #7b240c;
-  --bs-light-border-subtle: #666;
-  --bs-dark-border-subtle: #595959;
-  --bs-heading-color: #fff;
-  --bs-link-color: #97b2e9;
-  --bs-link-hover-color: #bacbf1;
-  --bs-link-color-rgb: 151, 178, 233;
-  --bs-link-hover-color-rgb: 186, 203, 241;
-  --bs-code-color: #ffd2f0;
-  --bs-border-color: #666;
-  --bs-border-color-translucent: rgba(255, 255, 255, 0.15);
-}
-
+/* stylelint-disable-line function-disallowed-list */
+/* stylelint-disable-line function-disallowed-list */
+/* stylelint-disable-line function-disallowed-list */
```

Comment:
Won't change anything for the release but this triple comment is interesting. Maybe we should avoid this kind of comments for stylelint.

Reply:
In the same spirit, `boosted.min.css` contains thinks like `// scss-docs-end collapse-transition\n\n// stylelint-disable function-disallowed-list\n// scss-docs-start aspect-ratios\n` which are useless. Let's see if they can be removed 🤷 in another issue/PR.